settings.gradle

```diff
@@ -180,8 +180,11 @@ include "beam-sdks-python-container"
 project(":beam-sdks-python-container").dir = file("sdks/python/container")
 include "beam-sdks-python-container-py3"
 project(":beam-sdks-python-container-py3").dir = file("sdks/python/container/py3")
+// Temporary version that will be deleted once we migrate to using the published version.
 include "beam-vendor-grpc-v1_13_1"
 project(":beam-vendor-grpc-v1_13_1").dir = file("vendor/grpc-v1_13_1")
+include "beam-vendor-grpc-1_13_1"
```

Comment:
Does the dependency version (`1_13_1`) need to be embedded in the project name/directory? That means upgrades will include renaming projects and moving a bunch of files around.

Could we just name it `beam-vendor-grpc`? Presumably only one version should exist at a time, and we can still embed the version number in the published artifact.

Reply:
It could be that we would have two versions in the codebase as stuff gets migrated from one version to another and following the pattern that Kenn was establishing. `git mv` is pretty easy to use for these cases and the cost of changing this isn't high if this becomes a big hassle.